### ISSUE TITLE
Is the validation target for the unsupervised part the test set ?

### ISSUE THREAD

**Issue by fehrlich** [dlcjfgmlnasa/NeuroNet#4]
https://github.com/dlcjfgmlnasa/NeuroNet/blob/3039645acbe359dc40884a3cfbf5f805717b55b5/pretrained/train.py#L181
```python
    def linear_probing(self, val_dataloader, eval_dataloader):
        self.model.eval()
        (train_x, train_y), (test_x, test_y) = self.get_latent_vector(val_dataloader), \
                                               self.get_latent_vector(eval_dataloader)
        pca = PCA(n_components=50)
        train_x = pca.fit_transform(train_x)
        test_x = pca.transform(test_x)

        model = KNeighborsClassifier()
        model.fit(train_x, train_y)

        out = model.predict(test_x)
        acc, mf1 = accuracy_score(test_y, out), f1_score(test_y, out, average='macro')
        self.model.train()
        return acc, mf1
```

I was wondering if this validation method is using the same eval test set that is used for fine tuning?
